js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/__fixtures__/AutoMaterializePolicyPage.fixtures.ts

```diff
@@ -285,9 +251,7 @@ export const Evaluations = {
           }),
           currentAutoMaterializeEvaluationId: 1000,
         }),
-        autoMaterializeAssetEvaluationsOrError: buildAutoMaterializeAssetEvaluationRecords({
-          records: [],
-        }),
+        assetConditionEvaluationRecordsOrError: buildAssetConditionEvaluationRecords() as any,
```

Comment:
Is it necessary to `any` these? It implies there's a problem somewhere if these had to be any'd...

Reply:
yeah the problem is that we our type generation doesn't work with polymorphic interfaces and MetadataEntry is polymorphic